sycl/unittests/pi/cuda/test_kernels.cpp

```diff
@@ -355,11 +361,11 @@ TEST_F(CudaKernelsTest, PIKernelArgumentSetTwiceOneLocal) {
 
   pi_program prog;
   pi_int32 binary_status = PI_SUCCESS;
-  ASSERT_EQ(
-      (plugin.call_nocheck<detail::PiApiKind::piProgramCreateWithBinary>(
-          context_, 1, &device_, nullptr,
-          (const unsigned char **)&threeParamsTwoLocal, &binary_status, &prog)),
-      PI_SUCCESS);
+  ASSERT_EQ((plugin.call_nocheck<detail::PiApiKind::piProgramCreateWithBinary>(
+                context_, 1, &device_, nullptr,
+                (const unsigned char **)&threeParamsTwoLocal, &binary_status,
+                nullptr, 0, &prog)),
+            PI_SUCCESS);
```

Comment:
Should we add a unittest with non-null values?

Reply:
I have added a test case `PICreateProgramAndKernelWithMetadata` to sycl/unittests/pi/cuda/test_kernels.cpp.